File: Lib/Telerik/Examples/QuickStart/ChartView/NullValues/FootballTeamStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Telerik.Examples.WinControls.ChartView.NullValues
{
    public class FootballTeamStats
    {
        string _name;
        string _logoPath;
        string _season;
        int? _wins;
        int? _draws;
        int? _losses;
        int? _goalDifference;
        int? _points;
        int? _position;

        public string Name
        {
            get
            {
                return this._name;
            }
            set
            {
                this._name = value;
            }
        }

        public string LogoPath
        {
            get
            {
                return this._logoPath;
            }
            set
            {
                this._logoPath = value;
            }
        }

        public string Season
        {
            get
            {
                return this._season;
            }
            set
            {
                this._season = value;
            }
        }

        public int? Wins
        {
            get
            {
                return this._wins;
            }
            set
            {
                this._wins = value;
            }
        }

        public int? Draws
        {
            get
            {
                return this._draws;
            }
            set
            {
                this._draws = value;
            }
        }

        public int? Losses
        {
            get
            {
                return this._losses;
            }
            set
            {
                this._losses = value;
            }
        }

        public int? GoalDifference
        {
            get
            {
                return this._goalDifference;
            }
            set
            {
                this._goalDifference = value;
            }
        }

        public int? Points
        {
            get
            {
                return this._points;
            }
            set
            {
                this._points = value;
            }
        }

        public int? Position
        {
            get
            {
                return this._position;
            }
            set
            {
                this._position = value;
            }
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/ChartView/NullValues/FootballTeamStats.vb
Imports System.Linq
Imports System.Text

Namespace Telerik.Examples.WinControls.ChartView.NullValues
	Public Class FootballTeamStats
		Private _name As String
		Private _logoPath As String
		Private _season As String
		Private _wins? As Integer
		Private _draws? As Integer
		Private _losses? As Integer
		Private _goalDifference? As Integer
		Private _points? As Integer
		Private _position? As Integer

		Public Property Name() As String
			Get
				Return Me._name
			End Get
			Set(ByVal value As String)
				Me._name = value
			End Set
		End Property

		Public Property LogoPath() As String
			Get
				Return Me._logoPath
			End Get
			Set(ByVal value As String)
				Me._logoPath = value
			End Set
		End Property

		Public Property Season() As String
			Get
				Return Me._season
			End Get
			Set(ByVal value As String)
				Me._season = value
			End Set
		End Property

		Public Property Wins() As Integer?
			Get
				Return Me._wins
			End Get
			Set(ByVal value? As Integer)
				Me._wins = value
			End Set
		End Property

		Public Property Draws() As Integer?
			Get
				Return Me._draws
			End Get
			Set(ByVal value? As Integer)
				Me._draws = value
			End Set
		End Property

		Public Property Losses() As Integer?
			Get
				Return Me._losses
			End Get
			Set(ByVal value? As Integer)
				Me._losses = value
			End Set
		End Property

		Public Property GoalDifference() As Integer?
			Get
				Return Me._goalDifference
			End Get
			Set(ByVal value? As Integer)
				Me._goalDifference = value
			End Set
		End Property

		Public Property Points() As Integer?
			Get
				Return Me._points
			End Get
			Set(ByVal value? As Integer)
				Me._points = value
			End Set
		End Property

		Public Property Position() As Integer?
			Get
				Return Me._position
			End Get
			Set(ByVal value? As Integer)
				Me._position = value
			End Set
		End Property
	End Class
End Namespace
